Staking pallet updates for the mark inactive collator feature (#791)
moonbeam-foundation/moonbeam-docs: builders/pallets-precompiles/pallets/staking.md

- **leaveCandidatesDelay**() - returns the number of rounds that must be waited before a scheduled request for a candidate to leave the candidate pool can be executed
- **leaveDelegatorsDelay**() - returns the number of rounds that must be waited before a scheduled request for a delegator to leave the set of delegators can be executed
- **maxBottomDelegationsPerCandidate**() - returns the maximum number of bottom delegations per candidate
- **maxCandidates**() - returns the maximum number of candidates allowed in the candidate pool
- **maxDelegationsPerDelegator**() - returns the maximum number of delegations per delegator  

> - **maxOfflineRounds**() - returns the number of rounds that must pass before a collator that has stopped producing blocks before they are marked as inactive

Reviewer: ```suggestion
- **maxOfflineRounds**() - returns the number of rounds that must pass before a collator that has stopped producing blocks is marked as inactive
```
